prow/release-commit.sh

```diff
@@ -40,7 +40,7 @@ BUILDER_SHA=a85593d26d52fcd77032bde5438cc04e7e9cb81c
 
 # Reference to the next minor version of Istio
 # This will create a version like 1.4-alpha.sha
-NEXT_VERSION=1.18
+NEXT_VERSION=1.19
```

Comment:
This value is already defined in the makefile. We should use the value there.

Reply:
Yeah, although format is different than the expected (`1.19-dev`) instead of just `1.19`. We can reuse the value in Makefile if we remove anything after the dash in this script. WDYT?

Reply:
We could remove after the dash in the script or add a new var without the -dev in the Makefile and use that, or even move the value to another file, and scrap that everywhere. I'm not as worried about where we get the value, but if we can limit the number of places we need to change at branch cut, it will simplify things in the future.